src/wallet/wallet.ts

```diff
@@ -275,24 +307,44 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     this.clearSensitiveData();
   }
 
-  generateCreateWalletAuthData(seed: string): CreateWalletAuthData {
-    const xpriv = walletUtils.getXPrivKeyFromSeed(seed, {passphrase: this.passphrase, networkName: this.network.name});
-    const xpub = walletUtils.getXPubKeyFromSeed(seed, {passphrase: this.passphrase, networkName: this.network.name});
-    const authXpub = HathorWalletServiceWallet.getAuthXPubKeyFromSeed(seed, {
-      passphrase: this.passphrase,
-      networkName: this.network.name,
-    });
+  generateCreateWalletAuthData(pinCode: string): CreateWalletAuthData {
+    let xpriv: bitcore.HDPrivateKey;
```

Comment:
Not a `xpriv`, we should change the variable name

Reply:
I think this shouldn't be defined here. It's only used inside each `if..else` block and inside each one has different meanings. In the first, it's the root private key and in the second is the private key in the account path.